### ISSUE TITLE
[papermerge-multilangocr] paperless-ngx broken with papermerge-multilangocr

### ISSUE THREAD

**Issue by Openhabber** [linuxserver/docker-mods#490]
Hello all,

lscr.io/linuxserver/paperless-ngx is broken if the mod linuxserver/mods:papermerge-multilangocr is used (yes, this mod is mentioned for paperless-ngx also and worked before).

It is broken in latest but also on 1.7.1, where it worked before. My only option at the moment is to use it without this mod.

docker-compose (relevant part)

```
services:
  paperless:
    image: lscr.io/linuxserver/paperless-ngx:latest #or 1.8.0, or 1.7.1
    container_name: paperless
    networks:
      <removed>
    environment:
      - DOCKER_MODS=linuxserver/mods:papermerge-multilangocr
      - OCRLANG="deu,eng"
      <removed>
    volumes:
      <removed>
    expose:
        - "8000"
    restart: unless-stopped
```

Log on startup:

```
[mod-init] Curl/JQ was not found on this system for Docker mods installing
Hit:1 http://archive.ubuntu.com/ubuntu focal InRelease
Get:2 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Get:3 http://archive.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Get:4 http://archive.ubuntu.com/ubuntu focal-updates/multiverse Sources [23.4 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal-updates/main Sources [646 kB]
Get:6 http://archive.ubuntu.com/ubuntu focal-updates/universe Sources [304 kB]
Get:7 http://archive.ubuntu.com/ubuntu focal-updates/restricted Sources [53.3 kB]
Get:8 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [30.2 kB]
Get:9 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [2,583 kB]
Get:10 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [1,606 kB]
Get:11 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1,186 kB]
Get:12 http://archive.ubuntu.com/ubuntu focal-security/main Sources [307 kB]
Get:13 http://archive.ubuntu.com/ubuntu focal-security/multiverse Sources [11.8 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-security/restricted Sources [53.1 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-security/universe Sources [122 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [27.5 kB]
Get:17 http://archive.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [1,497 kB]
Get:18 http://archive.ubuntu.com/ubuntu focal-security/main amd64 Packages [2,127 kB]
Get:19 http://archive.ubuntu.com/ubuntu focal-security/universe amd64 Packages [892 kB]
Fetched 11.7 MB in 1s (10.4 MB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  libcurl4 libjq1 libonig5
The following NEW packages will be installed:
  jq libjq1 libonig5
The following packages will be upgraded:
  curl libcurl4
2 upgraded, 3 newly installed, 0 to remove and 23 not upgraded.
Need to get 709 kB of archives.
After this operation, 1,062 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 curl amd64 7.68.0-1ubuntu2.13 [161 kB]
Get:2 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libcurl4 amd64 7.68.0-1ubuntu2.13 [235 kB]
Get:3 http://archive.ubuntu.com/ubuntu focal/universe amd64 libonig5 amd64 6.9.4-1 [142 kB]
Get:4 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 libjq1 amd64 1.6-1ubuntu0.20.04.1 [121 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 jq amd64 1.6-1ubuntu0.20.04.1 [50.2 kB]
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 5.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7, <> line 5.)
debconf: falling back to frontend: Teletype
dpkg-preconfigure: unable to re-open stdin: 
Fetched 709 kB in 1s (539 kB/s)
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 12222 files and directories currently installed.)
Preparing to unpack .../curl_7.68.0-1ubuntu2.13_amd64.deb ...
Unpacking curl (7.68.0-1ubuntu2.13) over (7.68.0-1ubuntu2.12) ...
Preparing to unpack .../libcurl4_7.68.0-1ubuntu2.13_amd64.deb ...
Unpacking libcurl4:amd64 (7.68.0-1ubuntu2.13) over (7.68.0-1ubuntu2.12) ...
Selecting previously unselected package libonig5:amd64.
Preparing to unpack .../libonig5_6.9.4-1_amd64.deb ...
Unpacking libonig5:amd64 (6.9.4-1) ...
Selecting previously unselected package libjq1:amd64.
Preparing to unpack .../libjq1_1.6-1ubuntu0.20.04.1_amd64.deb ...
Unpacking libjq1:amd64 (1.6-1ubuntu0.20.04.1) ...
Selecting previously unselected package jq.
Preparing to unpack .../jq_1.6-1ubuntu0.20.04.1_amd64.deb ...
Unpacking jq (1.6-1ubuntu0.20.04.1) ...
Setting up libcurl4:amd64 (7.68.0-1ubuntu2.13) ...
Setting up curl (7.68.0-1ubuntu2.13) ...
Setting up libonig5:amd64 (6.9.4-1) ...
Setting up libjq1:amd64 (1.6-1ubuntu0.20.04.1) ...
Setting up jq (1.6-1ubuntu0.20.04.1) ...
Processing triggers for libc-bin (2.31-0ubuntu9.7) ...
/sbin/ldconfig.real: /usr/local/lib/libjbig2enc.so.0 is not a symbolic link
[mod-init] Attempting to run Docker Modification Logic
[mod-init] Applying linuxserver/mods:papermerge-multilangocr files to container
s6-rc-compile: fatal: unable to read /etc/s6-overlay/s6-rc.d/init-mod-papermerge-multilangocr-add-package/type: No such file or directory
```

Thanks for your great work!

**Comment by Tsunami2056** [comment]
I can confirm I'm having the same behavior with this container version 

Linuxserver.io version:- ng-1.5.0-ls47 Build-date:- 2022-08-10T08:41:05+02:00

**Comment by j0nnymoe** [comment]
It's likely because the mods have been upgraded to a new version but the paperless-ngx images having had a rebuild since we updated our baseimages for these new version of the mods.

Aside to that, we're actually in the progress of deprecating our paperless-ng/x containers and we recommend switching to the official container provided by paperless-ngx: https://github.com/paperless-ngx/paperless-ngx

**Comment by Openhabber** [comment]
> It's likely because the mods have been upgraded to a new version but the paperless-ngx images having had a rebuild since we updated our baseimages for these new version of the mods.
> 
> Aside to that, we're actually in the progress of deprecating our paperless-ng/x containers and we recommend switching to the official container provided by paperless-ngx: https://github.com/paperless-ngx/paperless-ngx

Oh..may I ask why you deprecate paperless-ngx?

**Comment by Tsunami2056** [comment]
> Aside to that, we're actually in the progress of deprecating our paperless-ng/x containers and we recommend switching to the official container provided by paperless-ngx: https://github.com/paperless-ngx/paperless-ngx

Thanks for pointing this out, I had forgotten to migrate to the ngx container image.

**Comment by aptalca** [comment]
fixed in #491
